src/software/firmware/srcs/buzzer_control.cpp

```diff
@@ -0,0 +1,58 @@
+/******************************************************************************
```

Comment:
@pi-r-p When you have time, can we have some time for more explanation of this change? I know you already did a review with @dsferruzza and I miss it. Sorry.